src/clj/analysis_viewer/scripts.clj

```diff
@@ -0,0 +1,29 @@
+(ns analysis-viewer.scripts
+  (:require [api.config :as config]
+            [aws.s3 :as aws-s3]
+            [clojure.java.io :as io]
+            [clojure.string :as string]
+            [shared.utils :as shared-utils]))
+;; Run like
+;; clj -X analysis-viewer.scripts/ensure-maps-uploaded
+(defn ensure-maps-uploaded
```

Comment:
maybe we could use use the logging framework & facilities (timbre), the idea being if this will be run it will leave a trace in the AWS logs if we ever need it for post-mortems, debugging etc